operations/site.go

```diff
@@ -61,3 +64,38 @@ func CreateSite(cmd *cobra.Command, client *porcelain.Netlify, ctx context.Conte
 
 	return site, nil
 }
+
+func ChooseSite(client *porcelain.Netlify, ctx context.Context) (*models.Site, error) {
+	sites, err := client.ListSites(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+	nameToId := make(map[string]int)
+	for i, s := range sites {
+		fmt.Printf("[%d] %s (%s)\n", i+1, s.Name, s.ID)
```

Comment:
just a super minor thing, but, why are we showing the ID at all? let's not print it.